### ISSUE TITLE
Add examples for code reuse

### ISSUE THREAD

**Issue by na--** [k6io/docs#255]
It seems like with `scenarios`, people want to put different scenarios in different files and execute them separately, but also execute all of them at once. The requisite k6 issue for making this use case better is https://github.com/loadimpact/k6/issues/1342, but even now, people can achieve a lot by just having a `main` fail that imports and re-exports individual scenarios. Recent examples where this has popped up:
- https://community.k6.io/t/run-two-several-scenario-files-in-parallel-or-serially-using-single-file/1584
- https://github.com/k6io/k6/issues/1918

**Comment by simskij** [comment]
Good idea! We need more content around test project organization in general as almost all examples we currently have are single-file ones.

**Comment by na--** [comment]
Here is another recent example of a similar question: https://community.k6.io/t/to-have-a-shared-project-for-a-company-with-multiple-teams/1653

And a more complicated use case where someone had to wrap `handleSummary()` but leave everything else the same: https://github.com/k6io/k6/issues/1946

This latter case is interesting because I had to drop down to ES5.1, since the way k6 implements the ES6 `export` and `import` prevented me from using them instead of the ES5.1 `require()` and `exports`. So this might be worth documenting instead.
